packages/components/CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ### Enhancements
```

Comment:
Nit: Should this changelog entry be under `Bug Fixes` like the original PR?

Reply:
Well spotted — updated!